notebooks/tel_and_site/subsys_req_ver/dome/SITCOM-2016_dome_azimuth_endurance.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    
</pre><p>Please, add blank lines to help with code readability. For example, use them to create a block of code for ax1 and another block of code for ax2. Make sure you run <code>ruff format</code> on this file.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>